### ISSUE TITLE
Path to Stack binary error - but looks to be configured

### ISSUE THREAD

**Issue by yunti** [rikvdkleij/intellij-haskell#367]
plugin beta v37
IntelliJ IDEA 2018.3.1 (Ultimate Edition)
Build #IU-183.4588.61, built on December 4, 2018
JRE: 1.8.0_152-release-1343-b16 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.2
---------------

with haskell sdk configured
<img width="541" alt="image" src="https://user-images.githubusercontent.com/1826271/50249887-3a270b00-03d7-11e9-8dbe-244ded91fc81.png">

getting path to stack binary error
<img width="612" alt="image" src="https://user-images.githubusercontent.com/1826271/50249925-50cd6200-03d7-11e9-9ee0-08b89c049f45.png">

**Comment by rikvdkleij** [comment]
Thanks for reporting!

Is also SDK for project itself set?

Same problem with beta38?

Are there steps to reproduce this problem?

**Comment by rikvdkleij** [comment]
So check in Project settings if project SDK is set.

**Comment by yunti** [comment]
Yes the SDK is set in the project settings.  That was the first picture.

**Comment by rikvdkleij** [comment]
> Yes the SDK is set in the project settings. That was the first picture.

No, that is not how that page looks like. Check `Project Settings`/`Project`

Also problem with beta38?

**Comment by rikvdkleij** [comment]
You will have to check the SDK in `Project Settings` / `Project` in your screenshot.

**Comment by yunti** [comment]
I forgot to mention I'm now on beta 38 - (looks like we just cross posted)

**Comment by rikvdkleij** [comment]
I assume you have created a Runner. You will have to set an executable.

I will change the from so that field is mandatory to prevent this error.

**Comment by rikvdkleij** [comment]
To run something there must be an executable defined in your project's cabal file.

**Comment by yunti** [comment]
@rikvdkleij Thanks, however in the cabal file it points to the default /app/Main.hs but this isn't selectable in the runner's executable option - (nor is anything else unfortunately).

<img width="658" alt="image" src="https://user-images.githubusercontent.com/1826271/50772756-bdc17280-1286-11e9-8a4f-dbe4b3e1b3cd.png">

**Comment by rikvdkleij** [comment]
I can not reproduce your issue. Can you show me your whole Cabal file?

**Comment by yunti** [comment]
Thanks, got it to work by just starting a new project. Both cabal files are identical, so don't think the issue is with that. 

Non working project cabal file: https://gist.github.com/yunti/0e93e8bf9573129d36f6ada1f6e745ae
Working project cabal file: https://gist.github.com/yunti/95cd25ecb00856f9c931853313f2f09b

The difference in creating the project was in the working one, in the new project wizard I selected - `Import project from external model`, in the other one (test9) I didn't and selected `Create project from existing sources`.  If a user selects that on project setup, there doesn't seem to be a way to correct it into a working project - and the issues that I found occur.  
Hope that helps.
